### DISCUSSION TITLE
Octernship

### DISCUSSION THREAD

**Discussion by Shr11** [community/community#73821] [GitHub Education]
### Octernship 

Question:How to get selected by completing an octernship project ?

I got two octernship projects & I immediately accepted both of them but since they are completely different fields I've decided to work on one & I'm ready to learn, however in the projects repo it says past proven proficiency etc are considered.
So, does this reduce my chances of selection even if I submit the assignment?

(Please guide me on how to start working on octernship project & finish it successfully.)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Shr11 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by Shr11** [reply]
Thanks a lot! This was my first query so I wasn't really sure but I'll keep it in mind next time.
